storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -164,13 +164,13 @@
 (def COMMON-FIELDS [:emitted :transferred])
 (defrecord CommonStats [emitted transferred rate])
 
-(def BOLT-FIELDS [:acked :failed :process-latencies :executed :execute-latencies])
+(def BOLT-FIELDS [:acked :failed :process-latencies :executed :execute-latencies :deserialize-time])
 ;;acked and failed count individual tuples
-(defrecord BoltExecutorStats [common acked failed process-latencies executed execute-latencies])
+(defrecord BoltExecutorStats [common acked failed process-latencies executed execute-latencies deserialize-time])
 
-(def SPOUT-FIELDS [:acked :failed :complete-latencies])
+(def SPOUT-FIELDS [:acked :failed :complete-latencies :deserialize-time])
```

Comment:
This might be the point I didn't fully understand. 
In [here](https://github.com/apache/storm/blob/master/storm-core/src/clj/backtype/storm/daemon/executor.clj#L413-L429), all tuple to even spout can go through task-receiver right? I think these values are sent as bolt ones.